css/custom.css

```diff
@@ -386,4 +386,8 @@ li:hover .pin, li.hover .pin {
     .card {
         border-color: #e5e6e7;
     }
-}
\ No newline at end of file
+}
+
+#othersSection{
+    display: none;
```

Comment:
because there is too many there?

Reply:
Others are just all remaining uncategorized links. We could just display them but ultimately that's just a huge chunk of links almost nobody needs. Hiding them when not searching is a good compromise imo.

Reply:
But then we can also just remove the section no?

Reply:
The section is new, I created it specifically for searching through it :D

Reply:
If you want me to further explain the rationale behind my changes we can hop on discord anytime :)

Reply:
ah, upps I thought it was the SPECIAL section.